docs/dependencies-node-roadmap.md

```diff
@@ -0,0 +1,104 @@
+# Dependencies Node Roadmap
+
+This document describes, at a high level, the design and implementation of the Dependencies node with pointers to the important types.
+
+Let's start at the top and work our way down.
+
+## The CPS View of the Project Tree
+
```

Comment:
I know you call this out below, but it would be nice to call here that dependency node is actually CPS + GraphNodes.